scripts/install_ebpf.psm1

```diff
@@ -296,11 +296,10 @@ function Install-eBPFComponents
     }
 
 
-    # Copy ebpfapi.dll from installation directory to current directory.
-    # This ensures latest version of ebpfapi is used by the tests.
-    $ebpfApiDllPath = Join-Path $env:ProgramFiles "ebpf-for-windows\ebpfapi.dll"
-    Write-Log("Copying ebpfapi.dll from '$ebpfApiDllPath' to '$pwd'...")
-    Copy-Item -Path $ebpfApiDllPath -Destination $pwd -Force
+    # Refresh Path so EbpfApi.dll can be found.
+    $machinepath = [system.environment]::getenvironmentvariable("path", [system.environmentvariabletarget]::machine)
+    $userpath = [system.environment]::getenvironmentvariable("path", [system.environmentvariabletarget]::user)
+    $env:path = $machinepath + ";" + $userpath
```

Comment:
nit: should we append to the existing `$env.path` instead of replacing it with `$machinepath + ";" + $userpath`?

Reply:
I wasn't sure which is better.

If we don't make any manual changes to the path then this is more general -- if at some point the installation adds anything else to the system path this code will already catch all the changes.

On the other hand, currently `Program Files\ebpf-for-windows` is the only thing added to the path during installation, so I could just add that one directory to `$env:Path`, and then it wouldn't matter whether we make other path changes within the script.